site/content/en/docs/Reference/uninstalling_minikube.md

```diff
@@ -0,0 +1,72 @@
+---
+title: "Uninstall minikube"
+linkTitle: "Uninstall minikube"
+weight: 6
+date: 2019-08-18
+description: >
+  Reference on uninstalling minikube from your system completely.
+---
+
+# Uninstall minikube on Windows
+Following are the ways you can install minikube on Windows. Depending on how you installed minikube, please follow the guide appropriately.
+
+## Chocolatey

```

Comment:
I don't consider it neceserray, but if you want to get fancy, you can use the tabs widget that we use in the start docs:

```
{{% tabs %}}
{{% tab "Direct" %}}

Download and install minikube to /usr/local/bin:

```shell
 curl -LO https://storage.googleapis.com/minikube/releases/latest/minikube-linux-amd64 \
   && sudo install minikube-linux-amd64 /usr/local/bin/minikube
```
{{% /tab %}}
{{% tab "Debian/Ubuntu (deb)" %}}
```

I understand that you may be flying in the dark due to hugo panic'ing on Windows with our site.